packages/augur-sdk/src/connector/ws-connector.test.ts

```diff
@@ -0,0 +1,19 @@
+import { WebsocketConnector } from "./ws-connector";
```

Comment:
This isn't a proper jest test. I think @robert-e-davidson3 has thoughts on how to test this.

Reply:
+1